android/vendored/unversioned/@shopify/react-native-skia/android/build.gradle

```diff
@@ -137,8 +139,7 @@ android {
 
     externalNativeBuild {
         cmake {
-            path file('CMakeLists.txt')
-            version '3.10.2'
```

Comment:
we might want to remove the cmake version from the [versioned copy](https://github.com/expo/expo/blob/main/android/vendored/unversioned/%40shopify/react-native-skia/android/build.gradle#L141) as well, so that we don't require this cmake version to build.